pedalboard/io/PythonInputStream.h

```diff
@@ -219,4 +234,63 @@ class PythonInputStream : public juce::InputStream, public PythonFileLike {
   juce::int64 totalLength = -1;
   bool lastReadWasSmallerThanExpected = false;
 };
+
+/**
+ * A juce::InputStream subclass that fetches its
+ * data from a provided Python file-like object.
+ */
+class PythonMemoryViewInputStream : public PythonInputStream {
+public:
+  PythonMemoryViewInputStream(py::buffer bufferLike, py::object passedObject)
+      : PythonInputStream(bufferLike) {
+    info = bufferLike.request();
+    totalLength = info.size * info.itemsize;
+    repr = py::repr(passedObject).cast<std::string>();
+
+    if (py::hasattr(passedObject, "tell")) {
+      try {
+        offset = std::min(
+            std::max(0LL, passedObject.attr("tell")().cast<juce::int64>()),
+            totalLength);
+      } catch (py::error_already_set e) {
+        e.restore();
+      } catch (std::exception e) {
+        // If we can't call tell(), that's fine; just assume an offset of 0.
+      }
+    }
+  }
+
+  virtual std::string getRepresentation() override { return repr; }
+
+  virtual bool isSeekable() noexcept override { return true; }
+
+  virtual juce::int64 getTotalLength() noexcept override { return totalLength; }
+
+  virtual int read(void *buffer, int bytesToRead) noexcept override {
```

Comment:
You're mixing camel case and snake case here. In much of the rest of the code you use snake case for variable naming, shouldn't the same apply in this class?

Reply:
[Below](https://github.com/spotify/pedalboard/pull/291/files#diff-b41e5e23d58e8432233614cd153f7a1ebeab13d11d0c953141fdc1e7ea3a9266R292) I was looking at the definition for `py::buffer_info` and overfit to it. Fair point.